egs/swbd/s5c/local/chain/tuning/run_blstm_6j.sh

```diff
@@ -185,6 +186,7 @@ if [ $stage -le 13 ]; then
     --trainer.optimization.initial-effective-lrate 0.001 \
     --trainer.optimization.final-effective-lrate 0.0001 \
     --trainer.optimization.momentum 0.0 \
+    --trainer.optimization.alpha $alpha \
```

Comment:
When we actually commit this I'd want to have a new script in 'tuning', with results properly declared at the top.
What alpha value was used and do you have the output of compare_wer_general.sh for some suitable comparison?